modules/claims_api/lib/bgs_service/local_bgs.rb

```diff
@@ -188,8 +188,14 @@ def find_intent_to_file_by_ptcpnt_id_itf_type_cd(id, type)
       itf_type_cd = body.at 'itfTypeCd'
       itf_type_cd.content = type.to_s
 
-      make_request(endpoint: 'IntentToFileWebServiceBean/IntentToFileWebService',
-                   action: 'findIntentToFileByPtcpntIdItfTypeCd', body:, key: 'IntentToFileDTO')
+      response =
+        make_request(
+          endpoint: 'IntentToFileWebServiceBean/IntentToFileWebService',
+          action: 'findIntentToFileByPtcpntIdItfTypeCd',
+          body:
+        )
+
+      Array.wrap(response[:intent_to_file_dto])
```

Comment:
what was the intent with moving `key: 'IntentToFileDTO'` to be handled by `Array.wrap(response[:intent_to_file_dto])`? The logic in our controller would handle an array if needed. if we move to this I would rather [clean up the v1 controller](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/app/controllers/claims_api/v1/forms/intent_to_file_controller.rb#L57-L61) as well. Even, [v2 potentially](https://github.com/department-of-veterans-affairs/vets-api/blob/4e7c4d2893081bfbe6b6cca66f0bbb65fef6cf4b/modules/claims_api/app/controllers/claims_api/v2/veterans/intent_to_file_controller.rb#L29).

Reply:
I noticed that. But this change was only to preserve the exact behavior of what was already there in order to remove all special casing in `make_request` and this was already always returning an array. We could do wider cleanup and still have it be a pretty small change, but when I was doing this I wasn't entertaining any tangents, especially since this was already a prerequisite to my actual goal.

Reply:
TL;DR @FonzMP 
- the change to this function preserves its exact behavior for what it outputs
- i wanted to keep moving forward with POA stuff and not get sidetracked changing tangential code